blenderproc/python/types/URDFUtility.py

```diff
@@ -0,0 +1,898 @@
+from typing import Union, List
+import numpy as np
+from mathutils import Vector, Euler, Color, Matrix, Quaternion
+
+import bpy
+
+from blenderproc.python.utility.Utility import Utility
+from blenderproc.python.types.EntityUtility import Entity
+from blenderproc.python.types.MeshObjectUtility import MeshObject
+from blenderproc.python.types.BoneUtility import get_constraint, set_ik_constraint, \
+    set_ik_limits_from_rotation_constraint
+
+
+class Inertial(Entity):
```

Comment:
This file is quite long now. Can we maybe split it up into one file per class, so its easier to read?